tests/test_zarr.py

```diff
@@ -9,3 +9,12 @@ def test_exports() -> None:
 
     for export in __all__:
         getattr(zarr, export)
+
+
+def test_print_debug_info() -> None:
+    """
+    Ensure that print_debug_info does not raise an error
+    """
+    from zarr import print_debug_info
+
+    print_debug_info()
```

Comment:
to make this test a bit more meaningful, let's capture the output and assert that it matches our basic expectations. https://docs.pytest.org/en/stable/how-to/capture-stdout-stderr.html#accessing-captured-output-from-a-test-function

Reply:
I tested two outputs that should always be there. I didn't test every output because at some point that just becomes rewriting the fucntion